daemon/daemon_unix.go

```diff
@@ -1757,7 +1757,23 @@ func (daemon *Daemon) setupSeccompProfile() error {
 }
 
 func (daemon *Daemon) useShimV2() bool {
-	return cgroups.IsCgroup2UnifiedMode()
+	env := os.Getenv("MOBY_DISABLE_SHIM_V2")
```

Comment:
Instead of specifying a shim at all, can we have this be up to the containerd config?

Reply:
I think we write that config if containerd is started as child process, or did i recall incorrectly?

Reply:
This is true, however this seems like a fringe case. If someone wants to control this they can start containerd manually.